Darkmode.sh

```diff
@@ -74,9 +74,20 @@ solar() {
 	# Get Night Shift solar times (UTC)
 	riseT=$(/usr/bin/corebrightnessdiag nightshift-internal | grep nextSunrise | cut -d \" -f2)
 	setT=$(/usr/bin/corebrightnessdiag nightshift-internal | grep nextSunset | cut -d \" -f2)
+
+   	 # test 12 or 24 hour format
+   	 strig="M"
```

Comment:
```suggestion
	if [[ $riseT == *M* ]] || [[ $setT == *M* ]]; then
		formatT="%Y-%m-%d %H:%M:%S %p %z"
	else
		formatT="%Y-%m-%d %H:%M:%S %z"
	fi
```

Reply:
Suggest dropping the variable and just placing "M" in the test condition. 
Also added $setT and restructured to my standards.